src/database/migrations/0001_create_activity_logs.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE activity_logs (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  activity_type TEXT,
```

Comment:
Most or all of the TEXT fields could be VARCHAR. In postgres, they should work the same (assuming no length limit is specified). The latter should port better if ever needed.